### ISSUE TITLE
FAQ: "What do the exposure check logs show?" no longer matches Android UI

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#1028]
## Where to find the issue

**What do the exposure check logs show?**
https://www.coronawarn.app/en/faq/#keys_matches

**Was bedeutet die Überprüfung auf mögliche Begegnungen?**
https://www.coronawarn.app/de/faq/#keys_matches

## Describe the issue

Google changed the UI with v1.8 of the Exposure Notifications System. See [Exposure Notification release notes - v1.8, January 2021](https://developers.google.com/android/exposure-notifications/release-notes#v18_january_2021).

In Android there is no count "Exposure checks" any more.

The Hash value changes for every risk evaluation.

There are other inaccuracies in the English version noted in the next section.

## Suggested change

For Android:

The following section needs to be removed:

```
Exposure checks

This value does not reflect the number of encounters. 
Please tap one level further: This is where the checks of up to 15 days are logged (today plus the last 14 days). 
Selecting an entry leads to a detailed view with the corresponding time stamps and the number of matched keys.
```

The second sentence below does not make sense. It needs to be re-translated from the correct German.
```
Timestamp

Date and time at which the check happened. 
This is independent of the timestamp of potential exposures since every key stored in the downloaded files, a separate validity timestamp is stored.
```

The Android label is "App" not "Data Source" (English and German)
```
Data Source

Identifies the app that initiated the check. 
If you, for example, also installed Immuni or SwissCovid, their app identifiers could be noted there instead of 'Corona-Warn'
```

The Hash value changes every time. The second sentence should be deleted.
```
Hash

Checksum of the file containing the downloaded diagnosis keys. 
As long as the file content (diagnosis keys) does not change, this checksum will not change throughout the relevant time frame of 2 weeks.
```

**Comment by Ein-Tim** [comment]
It's very hard to maintain this entry, since the UI differs hugely between iOS and Android (see https://github.com/corona-warn-app/cwa-app-ios/issues/1731#issuecomment-749779893)

---

> The following section needs to be removed:
> 
> ```
> Exposure checks
> 
> This value does not reflect the number of encounters. 
> Please tap one level further: This is where the checks of up to 15 days are logged (today plus the last 14 days). 
> Selecting an entry leads to a detailed view with the corresponding time stamps and the number of matched keys.
> ```

Still valid for iOS, but has to reflect that there aren't only 15 entries (from 15 days) but also up to 6 from each day.


> The Android label is "App" not "Data Source" (English and German)
> 
> ```
> Data Source
> 
> Identifies the app that initiated the check. 
> If you, for example, also installed Immuni or SwissCovid, their app identifiers could be noted there instead of 'Corona-Warn'
> ```

iOS: "Data Source" / "Datenquelle" -> shouldn't be changed for iOS


> The Hash value changes every time. The second sentence should be deleted.
> 
> ```
> Hash
> 
> Checksum of the file containing the downloaded diagnosis keys. 
> As long as the file content (diagnosis keys) does not change, this checksum will not change throughout the relevant time frame of 2 weeks.
> ```


After selecting one Exposure check from [this](https://user-images.githubusercontent.com/67682506/112528973-0fda0a00-8da5-11eb-8885-8d7a2acd4a8b.jpeg) list, the first screen that's shown is [this](https://user-images.githubusercontent.com/67682506/112529041-21bbad00-8da5-11eb-8038-4f6343a700e5.jpeg), there's no hash at all. 
After selecting one file from the "NEW FILES" (German: "NEUE DATEIEN") section, you can [view](https://user-images.githubusercontent.com/67682506/112529110-35ffaa00-8da5-11eb-9125-55c056181f92.PNG) the Hash, the "Provided Key Count" (German: "Bereitgestellte Schlüssel"), the Data Source (German: "Datenquelle) and the Timestamp (German "Zeitstempel").

The hash values do never change once the hashes were downloaded, only new hashes appear when a risk check is carried out.

Hope that help's @MikeMcC399!

One more thing, let's not update #1027 but rather merge this quickly, and as soon as everything is figured out here we can open a PR for this issue, would that be ok for you?

**Comment by MikeMcC399** [comment]
@Ein-Tim 

Thanks for checking. Your screen shots show that the UI on iOS is very different to the Android one.

I think we need guidance from the Open Source Team. It would be simpler for readers to have separate FAQ articles for Android and for iOS if the contents are so different.

@dsarkar has already merged #1027 in any case.

**Comment by MikeMcC399** [comment]
@heinezen 
Could you please consider this issue in your review of the FAQs?

**Comment by heinezen** [comment]
Yes, I have added this to #971

**Comment by MikeMcC399** [comment]
As the release notes https://developers.google.com/android/exposure-notifications/release-notes#v18_january_2021 for Version 1.8 of the Google Exposure Notifications System describe, there were some changes in the Google UI made some months ago which are still not reflected in the FAQ https://www.coronawarn.app/en/faq/#keys_matches.

The following set of screenshots shows the English v1.8 (213016000) Exposure Notifications System user interface:

![Google_COVID19_UI_V1 8](https://user-images.githubusercontent.com/66998419/132947962-3d990bd4-350d-4c41-8be6-33e7a33a0217.jpg)

I will prepare a PR to make the necessary changes to https://www.coronawarn.app/en/faq/#keys_matches and https://www.coronawarn.app/de/faq/#keys_matches for the Google UI.

Maybe @Ein-Tim (or another iOS user) could help to cross-check if there are other iOS-specific changes which need to be made?

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Since my last comment (https://github.com/corona-warn-app/cwa-website/issues/1028#issuecomment-807304142) the UI under iOS didn't change, so everything I wrote there is still valid. 
How will you differentiate between iOS and Android? Maybe split the FAQ entry in two halfs, one for Android and one for iOS?

**Comment by MikeMcC399** [comment]
@Ein-Tim 
The FAQ has already been updated so that the Apple and Android terms are named side-by-side. I hope that is sufficient. Let me see as I work through it.

Actually I would change "Apple" to "iOS", otherwise "Android" would need to be "Google".

**Comment by MikeMcC399** [comment]
@Ein-Tim 
I've now submitted PR #1728. You're welcome to check if it is consistent with what you see on iOS.

I tried to make the minimum of changes to make it correct. If we wanted to put in more information, we would probably need to split it into two different articles for iOS and Android.

Especially at this stage in the pandemic and the maturity of the app, there is very little discussion on these numbers any more, so I don't think that any major extra effort would be worthwhile.
